pages/Guides/supernode-program/docs.en.md

```diff
@@ -6,35 +6,76 @@ taxonomy:
 
-The NEM network has been designed from the very beginning with the goal in mind that any light client can securely connect to and use any server safely to make any transaction. Supernodes are expected to be high performance and reliable nodes. They are regularly tested on their bandwidth, chain height, chain parts, computing power, version, ping, and responsiveness to make sure that they are performing to high standards. If they meet all these requirements, they are randomly given rewards.
+Validators are rewarded daily for passing all rounds of tests. Each day are executed 4 rounds (every 6 hours) of tests. In order to complete round node must pass following tests:
 
-## Steps to Enroll and Participate in Supernodes
+**Chain Height**
+The chain height test checks that a node reports the correct height via the ``/chain/height API``. It retrieves a node’s height and compares it to the reference node’s height. 
 
-- **Step 1**: Download and run NEM Wallet from [the GitHub releases page](https://github.com/NemProject/NanoWallet/releases).
+This test passes if the node reports a height that is no more than four blocks behind the reference node’s height.
 
-  You can find tutorials on how to use NEM Wallet [here](https://nemproject.github.io/nem-docs/pages/Guides/nanowallet/docs.en.html).
+**Chain Part**
+The chain part test checks that a node can serve a random subset of blocks close to the end of the chain via the ``/chain/blocks-after`` API. First, it retrieves a node’s self reported height. Next, it requests a random number of blocks (between 60 and 100) from the node. Finally, it calculates the hash of each block and then a single composite hash over those hashes. The same composite hash is calculated over blocks retrieved from the reference node.
 
-- **Step 2**: Make an account and have at least 10,010 XEM.
+This test passes if all returned blocks are verifiable with proper signatures and the calculated composite hashes from the node and reference node match.
 
-  10,000 XEM to be able to participate in the Program, and ~10 XEM to be able to pay the fees for activating delegated harvesting and sending the enrollment message. (You have to re-enroll in the Program every month - 10 XEM should be enough to cover fees for at least a year.)
+**Computing Power**
+The computing power test checks that a node is running on sufficiently strong hardware. The master calls a node’s ``nr/computing-power`` API with a random hash. The node interprets that hash as a private key and derives a public key from it. It then interprets that new public key as a private key and derives another public key from it. This is repeated 10000 times and the final public key is returned to the master.
```

Comment:
@orisdorch 
- controller (replaced from the master after Wayon suggestion) is an application that performs tests.
- for some tests controller is using a reference node. For example to check if the testing node height is the latest one controller needs to fetch the first height from the reference node to compare.

Do you think for clarity we should replace all node -> testing node?

Reply:
No, but there are a few moving parts so it might be nice to basically add what you just provided here to help guide the reader on the terminology 
i.e. "Node testing is facilitated by the controller. A reference node is also used for localization and/or comparative purposes in some tests."